### PR TITLE
Refine UI for Non-premium Popup Panel 

### PR DESCRIPTION
**Pull request by codemist**
There were some strange flexbox issues with the popup panel, particularly with the image placements of the non-premium view. This PR should iron out any of the inconsistencies. Link below shows how the UI looks now.

Preview: https://drive.google.com/file/d/1Tpw4OvOhBuHogXjA1-4wdaS0JP0gX5wE/view?usp=sharing

### CODE REVIEW

**Comment by codemist** [comment]
No how I managed to push a commit titled 'git status', it's supposed to 'remove max-height' 🤦

**Review by Vinnl — APPROVED** [review]
Nice cleanup, looks much better here!

File: src/css/popup.css [hunk 1]
```diff
@@ -508,34 +507,11 @@ onboarding-panel p {
...
-  right: 30px;
+  padding: 16px;
 }
 
 .maxAliasesPanel .onboarding-img {
-  position: absolute;
-  right: 0;
-  left: 0;
-  top: 44px;
+  padding: 0px;
```
Comment (maxxcrawford) [inline src/css/popup.css:514]: nit: You can drop the `px` from here!

**Review by maxxcrawford — APPROVED** [review]
Double-approving this cleanup. Nice one, @codemist!

File: src/css/popup.css (hunk 1 above)
Comment (codemist) [reply src/css/popup.css]: done! @maxxcrawford

**Pull request merged**